roles/install_java/molecule/resources/verify.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Verify correct java version is selected
+  hosts: all
+  tasks:
+    - name: Get Java version
+      ansible.builtin.shell:
+        cmd: |
+          set -o pipefail
+          java -version 2>&1 | grep version | awk '{print $3}' | sed 's/"//g'
```

Comment:
the output from `java -version` is:
```
openjdk version "1.8.0_402"
OpenJDK Runtime Environment (build 1.8.0_402-b06)
OpenJDK 64-Bit Server VM (build 25.402-b06, mixed mode)
```
The output from the command on line 9 is:
```
1.8.0_402
```